docs/application/native/guides/machine-learning/machine-learning-inference.md

```diff
@@ -298,6 +298,120 @@ Followings are the available elements:
     char pipeline[] = "videotestsrc is-live=true ! videoconvert ! tensor_converter ! output-selector name=outs outs.src_0 ! tensor_sink name=sink0 async=false outs.src_1 ! tensor_sink name=sink1 async=false"
     ```
 
+- **If**
+
+    The if (tensor_if) element allows creating conditional branches based on tensor values. For example, you may skip frames if there is no object detected with high confidence. The input and output stream data type is either `other/tensor` or `other/tensors`.
+
+    - **Properties**
+      - `compared-value`: Specifies the compared value and is represented as operand 1 from input tensors.
+        - A_VALUE: Decided based on a single scalar value.
+	    - TENSOR_AVERAGE_VALUE: Decided based on an average value of a specific tensor.
+	    - CUSTOM: Decided based on a user-defined callback.
+      - `compared-value-option`: Specifies an element of the nth tensor or you can pick one from the tensors.
+	    - [C][W][H][B],n (used for A_VALUE of the compared-value. e.g., 0:1:2:3,0 means [0][1][2][3] value of first tensor)
+        - nth tensor (used for TENSOR_AVERAGE_VALUE of the compared-value to specify which tensor are used.)
```

Comment:
```suggestion
        - nth tensor: used for TENSOR_AVERAGE_VALUE of the compared-value, and specifies which tensor is used.
```
Please check if the suggestion is technically fine, if not please rephrase it.

Reply:
Thanks. Updated.